app/assets/stylesheets/components/header.scss

```diff
@@ -1,25 +1,19 @@
 @import '../config/import';
 
-.top-bar {
+.crayons-header {
```

Comment:
I renamed our header class to be `crayons-header` - this is a weird exception: this type of components are usually app oriented, but for headers it's likely we will want to stay consistent no matter what app so here it is..

Reply:
I like this update.